packages/vm/Cargo.toml

```diff
@@ -7,6 +7,15 @@ description = "VM bindings to run cosmwams contracts"
 repository = "https://github.com/CosmWasm/cosmwasm/tree/main/packages/vm"
 license = "Apache-2.0"
 
+[package.metadata.release]
+pre-release-replacements = [
+    { file="../../CHANGELOG.md", search="Unreleased", replace="{{version}}" },
+    { file="../../CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly=1 },
+    { file="../../CHANGELOG.md", search="ReleaseDate", replace="{{date}}" },
+    { file="../../CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [Unreleased] - ReleaseDate", exactly=1 },
+    { file="../../CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/CosmWasm/cosmwasm/compare/{{tag_name}}...HEAD", exactly=1 },
```

Comment:
I think there's something not quite right here. I get this diff when running `cargo release 2.0.2`:
```diff
<!-- next-url -->
+[Unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.2...HEAD
 
-[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...HEAD
+[unreleased]: https://github.com/CosmWasm/cosmwasm/compare/v2.0.1...v2.0.2
```
The spacing would need to be fixed and the `unreleased` in the changelog needs to become `Unreleased` (or the regex case-insensitive).

Reply:
I thought I fixed this already because I ran into the same issue. Ugh, gotta check where that got lost